pkg/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -126,18 +126,23 @@ func NewNodePerformance(assetsDir string, profile *performancev2.PerformanceProf
 
 		for _, device := range profile.Spec.Net.Devices {
 			devices = make([]string, 0)
-			if device.InterfaceName != nil {
-				devices = append(devices, *device.InterfaceName)
+			if device.DeviceID != nil {
+				devices = append(devices, "^ID_MODEL_ID="+*device.DeviceID)
 			}
 			if device.VendorID != nil {
-				devices = append(devices, *device.VendorID)
+				devices = append(devices, "^ID_VENDOR_ID="+*device.VendorID)
 			}
-			if device.DeviceID != nil {
-				devices = append(devices, *device.DeviceID)
+			if device.InterfaceName != nil {
+				deviceNameAmendedRegex := strings.Replace(*device.InterfaceName, "*", ".*", -1)
```

Comment:
why is this needed again? e.g. why can't we just expect `foobar.*` directly?

Reply:
The intended wildcards to use in PAO are * and !  , I need to convert these under the hood so they match a python regular expression format